pkg/operator/starter.go

```diff
@@ -280,14 +280,27 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		return err
 	}
 
+	apiServerConfig, err := configClient.ConfigV1().APIServers().Get(ctx, "cluster", metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	// TODO(4.7): switch over to secure access-token logging by default and delete old non-sha256 tokens
+	auditPolicyPathGetterWithAccessTokenLogs := func(profile string) (string, error) {
+		if apiServerConfig.Annotations["oauth-apiserver.openshift.io/secure-oauth-token-storage"] == "true" {
```

Comment:
what if the user removes the annotation from the APIServer object or set it to false? The `apiServerConfig` object is stale once it is changed by the user. May be I am misunderstanding the flow here?

Reply:
it will now redeploy with the old policy.